mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule ElixirAuthGoogle.MixProject do
   use Mix.Project
 
   @description "Minimalist Google OAuth Authentication for Elixir Apps"
-  @version "1.0.3"
+  @version "1.0.4"
```

Comment:
Let's make this match the version in `elixir-auth-github` considering the feature parity. 💭 
```suggestion
  @version "1.1.0"
```